configs/components/facter.rb

```diff
@@ -171,6 +171,10 @@
     end
   end
 
+  # In PE, aio_agent_version is distinguished from aio_agent_build by not including the git sha.
+  # Strip it off; this should have no impact on final releases, as git sha would not be included.
+  aio_agent_version = settings[:package_version].match(/^\d+\.\d+\.\d+(\.\d+){0,2}/).to_s
```

Comment:
I tested by adding `puts "#{aio_agent_version}"; exit`, then run a build (which would exit before starting to build things).